tests/integration/helpers_deployments.py

```diff
@@ -119,46 +119,61 @@ async def get_unit_hostname(ops_test: OpsTest, unit_id: int, app: str) -> str:
     return hostname.strip()
 
 
+async def unit_from_raw(
+    ops_test: OpsTest,
+    unit_name: str,
+    raw_unit: Dict[str, Any],
+    app_name: str,
+    raw_app: Dict[str, Any],
+    subordinate: bool = False,
+) -> Unit:
```

Comment:
I don't think this this function needs to be async.

nit: Perhaps, for consistency over the order and name of arguments (`app` is set as the first argument after ops_test in all test helpers)  
```suggestion
def _get_unit(
    ops_test: OpsTest,
    app: str,
    raw_app: dict[str, Any],
    unit_name: str,
    raw_unit: dict[str, Any],
    subordinate: bool = False,
) -> Unit:
```

Reply:
Actually, I made it async since there is an `await get_unit_hostname(ops_test, unit_id, app_name)` in the function.